src/BloomExe/MsTsfInterfaces.cs

```diff
@@ -0,0 +1,352 @@
+﻿// Copyright (c) 2013 SIL International
+// This software is licensed under the MIT License (http://opensource.org/licenses/MIT)
+#if !__MonoCS__
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Unmanaged.TSF
```

Comment:
I just copied the entire file from Palaso, so I can't say if it should be that way.